site/src/components/input.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Input
+description: The input component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D506%253A1176
+---
+
+# Input component
```

Comment:
Most H1s are just the name - only "Input" here, not "Input component". Think the shorter version is better. Same problem with `textarea`

Reply:
Updated